pkg/cloudprovider/provider/azure/provider.go

```diff
@@ -121,7 +121,7 @@ var imageReferences = map[providerconfigtypes.OperatingSystem]compute.ImageRefer
 	providerconfigtypes.OperatingSystemCentOS: {
 		Publisher: to.StringPtr("OpenLogic"),
 		Offer:     to.StringPtr("CentOS"),
-		Sku:       to.StringPtr("7-CI"), // https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init
+		Sku:       to.StringPtr("7_9"), // https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init
```

Comment:
Should we re-enable CentOS tests on Azure now that CentOS should work?

https://github.com/kubermatic/machine-controller/blob/f1979a1d104fb88d3286307de576ec33c30a5c09/test/e2e/provisioning/all_e2e_test.go#L564-L565